druid/Cargo.toml

```diff
@@ -117,4 +117,4 @@ required-features = ["svg", "im", "image", "png"]
 
 [[example]]
 name = "wgpu_view"
-required-features = ["wgpu", "bytemuck", "cgmath"]
+required-features = ["bytemuck", "cgmath", "wgpu_view"]
```

Comment:
Isn't the required feature `wgpu` (the external crate) and not `wgpu_view` (the alias name used inside Druid)?

Reply:
I don't know! I needed this to get it to compile as far as I remember, but could be a mistake. Will try it without that

Reply:
<pre>
 ~/dev/rust/druid/druid on 🌱 rjw_wgpu_view [!?] is 📦 v0.7.0 via 🦀 v1.49.0 
➜ cargo run --example wgpu_view                                
error: target `wgpu_view` in package `druid` requires the features: `bytemuck`, `cgmath`, `wgpu`
Consider enabling them by passing, e.g., `--features="bytemuck cgmath wgpu"`

~/dev/rust/druid/druid on 🌱 rjw_wgpu_view [!?] is 📦 v0.7.0 via 🦀 v1.49.0 
➜ cargo run --example wgpu_view --features wgpu,bytemuck,cgmath
   Compiling druid v0.7.0 (/Users/robertwittams/dev/rust/druid/druid)
error[E0432]: unresolved imports `druid::widget::WgpuRenderer`, `druid::widget::WgpuView`
  --> druid/examples/wgpu_view.rs:22:42
   |
22 | use druid::widget::{Align, Label, Split, WgpuRenderer, WgpuView};
   |                                          ^^^^^^^^^^^^  ^^^^^^^^ no `WgpuView` in `widget`
   |                                          |
   |                                          no `WgpuRenderer` in `widget`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: could not compile `druid`

To learn more, run the command again with --verbose.

</pre>